corehq/apps/reports/standard/monitoring.py

```diff
@@ -1737,12 +1738,12 @@ def avg_datespan(self):
             avg_datespan.startdate = datetime.datetime(1900, 1, 1)
         return avg_datespan
 
-    def _report_data(self):
+    def _report_data(self, users_to_iterate, user_ids):
         export = self.rendered_as == 'export'
         avg_datespan = self.avg_datespan
 
-        case_owners = _get_owner_ids_from_users(self.users_to_iterate)
-        user_ids = self.user_ids
+        case_owners = _get_owner_ids_from_users(users_to_iterate)
+        user_ids = user_ids
```

Comment:
I don't think this line does anything (but "user_ids is user_ids" does kinda remind me of [this pretty hilarious 80's song](https://www.youtube.com/watch?v=rQSmfR1_SVE) :joy: ).

Reply:
😂 good catch